app/app.html

```diff
@@ -10,12 +10,17 @@
     <link href="../node_modules/angular-color-picker/angular-color-picker.css" rel="stylesheet" type="text/css">
     <link href="../node_modules/codemirror/lib/codemirror.css" rel="stylesheet" type="text/css">
     <style></style>
-	
+
     <script src="../node_modules/codemirror/lib/codemirror.js"></script>
     <script src="../node_modules/codemirror/mode/css/css.js"></script>
     <script src="../node_modules/codemirror/mode/javascript/javascript.js"></script>
     <script src="../node_modules/codemirror/mode/xml/xml.js"></script>
     <script src="../node_modules/codemirror/mode/htmlmixed/htmlmixed.js"></script>
+    <script async
+    id="__bs_script__"
+    src="http://localhost:8081/browser-sync/browser-sync-client.js?v=2.27.5">
+    </script>
```

Comment:
this means this script will attempt to load even in non-development environments?  that doesn't seem right to me.